src/View/Cell.php

```diff
@@ -259,15 +260,17 @@ protected function _cacheConfig($action)
      * This is because PHP will not allow a __toString() method to throw an exception.
      *
      * @return string Rendered cell
+     * @throws \Error Include error details for PHP 7 fatal errors.
      */
     public function __toString()
     {
         try {
             return $this->render();
         } catch (Exception $e) {
             trigger_error(sprintf('Could not render cell - %s [%s, line %d]', $e->getMessage(), $e->getFile(), $e->getLine()), E_USER_WARNING);
-
             return '';
+        } catch (Error $e) {
+            throw new Error(sprintf('Could not render cell - %s [%s, line %d]', $e->getMessage(), $e->getFile(), $e->getLine()));
```

Comment:
@chinpei215 no it's not the return value. A fatal error could be a language syntax error or other unrecoverable errors. Such as ``$x = 1 &&&&& 2;`` where `&&&&&` is not a valid operator.

These types of errors were not catchable in previous versions of PHP, but in PHP 7 they are now declared using the base class `Error`. They should not be caught as this would change the behavior of the code compared to older versions of PHP, but with that said when this happens in ``__toString()`` the error message itself is not informative. So this PR adds the `could not render` message to the fatal error.

Since older versions of PHP do not throw `Error` as they will just crash at the execution point. The ``__toString()`` method in those version never returns, and since the new catch I added is after `Exception` it will not interfere with the way it use to work.